edx_sga/tests.py

```diff
@@ -389,6 +389,60 @@ def test_get_staff_grading_data(self):
         self.assertEqual(assignments[1]['annotated'], None)
         self.assertEqual(assignments[1]['comment'], u'')
 
+    def test_student_module_created(self):
+        def make_student_without_adding_module(name, **state):
```

Comment:
This looks like you are using the same pattern as the PR.  This appears to be the same as `make_student`, why not just add `make_state=True` to `make_student` and use that flag to do state handling?  Also, even if this needs to be a totally different function (because I'm misreading it), it wouldn't need to be a closure here because it is only called once and doesn't use any variables from the outer scope.